### ISSUE TITLE
Namespace 'System' could not be found

### ISSUE THREAD

**Issue by kbmuri** [Microsoft/Windows-universal-samples#300]
Greetings. I'm new to GitHub and learning Windows Universal, but have been programming Windows professionally since Windows began in the middle 1980's. Some DOS even before that, and up to and including Windows Phone 2010 and Windows 10 apps using VS express 2013. I'm a complete master of this domain. Currently trying to get back to leading-edge with VS 2015 and universal apps, including perhaps Xamarin cross-platform eventually.

I was interested in sampling your Windows Universal Samples snippets, particularly the AudioGraph concepts in the AudioCreation sample code. I downloaded the zip and extracted the entire archive, per the instructions. I opened the cs project of interest. Without modifying the code, it contains hundreds of errors revolving around an inability to resolve the fundamental "System" environment. A picture is worth 1000 words:

![audiocreationproject](https://cloud.githubusercontent.com/assets/17327534/13164957/007a755c-d687-11e5-8ca3-e9bee732df94.png)

I'm able to create and develop other windows universal apps from scratch with this environment with no problems "using System". I eventually created a blank project and pulled code from your sample into it and built an AudioGraph sample and have been experimenting with it but have run into other issues outside the scope of this specific request. I sure would like to see your example working as advertised, to compare my issues against a working model.

Please tell me what I did wrong to get the errors pictured above. Thank you.

**Comment by Adaok** [comment]
You can go on References, in the Nuget Manager for uninstall and re-install the Microsoft.NETCore.UniversalWindowsPlatform package for solve your problem.

I hope it will help you.

**Comment by kbmuri** [comment]
Thank you Adaok, That worked. Upon reinstalling the platform, I was prompted to accept the license terms for the installed packages. Soon as I checked "I Accept", all of the errors went away. Presumably I had to set a flag somewhere in this solution that signified I accepted the license agreement. Uninstalling and reinstalling apparently made that happen under the covers somewhere. Not intuitive but It works now.
